Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -538,6 +538,32 @@ theorem mul_neg_geom_series (x : R) (h : ‖x‖ < 1) : ((1 - x) * ∑' i : ℕ,
   rw [← mul_neg_geom_sum, Finset.mul_sum]
 #align mul_neg_geom_series mul_neg_geom_series
 
+theorem geom_series_succ (x : R) (h : ‖x‖ < 1) : ∑' i : ℕ, x ^ (i + 1) = ∑' i : ℕ, x ^ i - 1 := by
+  apply symm
+  rw [sub_eq_iff_eq_add, tsum_eq_zero_add (NormedRing.summable_geometric_of_norm_lt_one x h),
+    pow_zero, add_comm]
+
+theorem geom_series_mul_shift (x : R) (h : ‖x‖ < 1) :
+    x * ∑' i : ℕ, x ^ i = ∑' i : ℕ, x ^ (i + 1) := by
+  refine' tendsto_nhds_unique
+      ((NormedRing.summable_geometric_of_norm_lt_one x h).hasSum.mul_left x).tendsto_sum_nat _
+  have : Tendsto (fun n : ℕ => ∑ i in Finset.range (n + 1), x ^ i - 1) atTop
+      (𝓝 (∑' i : ℕ, x ^ (i + 1))) := by
+    rw [geom_series_succ x h]
+    apply Tendsto.sub_const
+    simp only [sum_range_succ]
+    have hp : tsum (fun (i : ℕ) => x^i) = tsum (fun (i : ℕ) => x^i) + 0 := (add_zero _).symm
+    rw [hp]
+    apply Tendsto.add ((NormedRing.summable_geometric_of_norm_lt_one x h).hasSum.tendsto_sum_nat)
+      (tendsto_pow_atTop_nhds_zero_of_norm_lt_one h)
+  convert this
+  rw [@geom_sum_succ _ _ x, mul_sum, add_sub_cancel_right]
```

Comment:
```suggestion
  rw [← Summable.tsum_mul_left]
  conv => enter [1, 1, i]; rw [← _root_.pow_succ']
  exact NormedRing.summable_geometric_of_norm_lt_one _ h 
```
(Also, `refine'` is being phased out...)

Reply:
Or in one  line:
```lean
  simp_rw [← (NormedRing.summable_geometric_of_norm_lt_one _ h).tsum_mul_left, ← _root_.pow_succ']
```

Reply:
Oh that's much nicer, thank you!